compiler-rt/lib/interception/interception_win.cpp

```diff
@@ -658,7 +681,6 @@ static size_t GetInstructionSize(uptr address, size_t* rel_offset = nullptr) {
   }
 
   switch (0x00FFFFFF & *(u32*)address) {
-    case 0x07c1f6:    // f6 c1 07 : test cl, 0x7
```

Comment:
Why are some old existing cases removed?

Reply:
That `f6 c1 07 : test cl, 0x7` got moved into the generic form `F6 C1 XX : test cl, XX`.
I tried to document this by moving it into separate commits before the "big" one.

Reply:
Ah, I see - so this should ideally be reviewed as a series of individual patches, rather than looking at the total result of this PR. The policy within llvm is (IMO unfortunately) that PRs are merged with "squash and merge" only, so the incoming distinction between individual commits is lost on merge, and most reviewers probably don't look at it with a per-commit view (unless told to).